tests/README.md

```diff
@@ -0,0 +1,86 @@
+# Testing dbt-tidb
+
+## Overview
+
+Here are the steps to run the tests:
+1. Set up
+2. Set environment variables
+3. Run TiDB
+4. Run tests
+
+## Set up
+
+Make sure you have python environment
```

Comment:
Does it require some particular python version? such as python3.8.

Reply:
I will add it in set up